### PR TITLE
Content: Add Cyberia to sponsor list

### PR DESCRIPTION
**Pull request by davidebeatrici**

### CODE REVIEW

**Comment by Krzmbrzl** [comment]
Did you/we already receive the hardware?

**Comment by davidebeatrici** [comment]
It was shipped and I have the tracking number, I should receive it next week.

**Review by Krzmbrzl — APPROVED** [review]

**Comment by toby63** [comment]
:thinking: Sry for intervening, but one suggestion from me: I think the sponsors list would be better, if it is less wordy.

File: hugo/content/sponsors.md [hunk 1]
```diff
@@ -2,7 +2,24 @@
 title: Sponsors
 ---
 
-Our infrastructure is made possible thanks to the help of the sponsors that supported us over the years by providing their services for free.
+Our infrastructure is made possible thanks to the help of the following sponsors:
+
+## Cyberia
+
+<img src="/images/lain.png" alt="Let's all love Lain!" style="float:right; width: 380px; max-width: 40vw">
```
Comment (Kissaki) [inline hugo/content/sponsors.md:9]: This change adds an image reference, but contains no added image file.

**Review by Kissaki — CHANGES_REQUESTED** [review]

**Comment by Kissaki** [comment]
> 🤔 Sry for intervening, but one suggestion from me: I think the sponsors list would be better, if it is less wordy (no offense).

I agree in general. I also think the tone could be more professional/neutral. But looking at the current sponsors page every section has quite a bit of content, so I think the text in this change is fine.

**Comment by davidebeatrici** [comment]
If the page ends up being too long we can just make each section expandable.

File: hugo/content/sponsors.md [hunk 2]
```diff
@@ -2,7 +2,24 @@
...
+
+The name should ring a bell if you have watched the masterpiece that is Serial Experiments Lain!
```
Comment (Kissaki) [inline hugo/content/sponsors.md:11]: Note: People unfamiliar with it will not know what that even is - that it is an anime.

**Review by Kissaki — APPROVED** [review]
Thought: I would prefer the sponsors page being a folder and the images being contained within it. Atm we have cross dependencies to static images. But we do not have to do that here in this PR.

Question: Is that the kind of image we want to put there?

Otherwise LGTM

File: hugo/content/sponsors.md (hunk 2 above)
Comment (davidebeatrici) [reply hugo/content/sponsors.md:11]: I mention it right below though:
> [Cyberia](https://cyberia.club) is managed by fans who decided to name their awesome collective after an important place in the anime.

**Comment by davidebeatrici** [comment]
> Thought: I would prefer the sponsors page being a folder and the images being contained within it. Atm we have cross dependencies to static images. But we do not have to do that here in this PR.

I agree.

> Question: Is that the kind of image we want to put there?

Cyberia's official logo is work in progress, the image I used is actually made by me: I removed the background from a scene using Krita.

File: hugo/content/sponsors.md (hunk 2 above)
Comment (toby63) [reply hugo/content/sponsors.md:11]: Maybe you could change it vice-versa. 
1. Explaning what Cyberia is, a hacker collective it seems.
2. Explaning that it refers to the anime.

**Comment by toby63** [comment]
**Logo:** What about using the ASCII logo from their website instead? https://cyberia.club/
The picture looks a bit scary imho.

**Text:** As Kissaki suggested I would make it sound a bit more neutral and make it shorter.

**Comment by davidebeatrici** [comment]
I received the laptop on Monday, it's been working perfectly.

**Pull request merged**